spec/models/entity_spec.rb

```diff
@@ -342,6 +342,15 @@ class AMapper
         end
 
         describe 'failures' do
+          context 'timeout error' do
+            before { allow(connec_client).to receive(:get).and_raise(Net::OpenTimeout) }
+
+            it 'retries and raise the error' do
+              expect {subject.get_connec_entities(nil)}.to raise_error(Net::OpenTimeout)
+              expect(connec_client).to have_received(:get).exactly(3).times
```

Comment:
This is 3 because of Retriable's default configuration?

Reply:
Good point, it's the default config. I'll make it explicit.